apps/els_lsp/src/els_completion_provider.erl

```diff
@@ -183,12 +209,188 @@ find_completions( Prefix
             ++ definitions(Document, POIKind, ExportFormat)
             ++ els_snippets_server:snippets()
       end;
-    _ ->
+    [] ->
+      %% Token parsing fails when triggering completion inside a string literal
+      case Prefix of
+        <<"-include_lib(\"">> ->
+          [item_kind_file(Path) || Path <- paths_include_lib()];
+        <<"-include(\"">> ->
+          [item_kind_file(Path) || Path <- paths_include(Document)];
+        _ ->
+          []
+      end;
+    Tokens ->
+      ?LOG_DEBUG("No completion found. [prefix=~p] [tokens=~p]",
+                 [Prefix, Tokens]),
       []
   end;
 find_completions(_Prefix, _TriggerKind, _Opts) ->
   [].
 
+%%=============================================================================
+%% Attributes
+%%=============================================================================
+-spec attributes() -> items().
+attributes() ->
+  [ snippet(attribute_behaviour)
+  , snippet(attribute_callback)
+  , snippet(attribute_compile)
+  , snippet(attribute_define)
+  , snippet(attribute_dialyzer)
+  , snippet(attribute_export)
+  , snippet(attribute_export_type)
+  , snippet(attribute_if)
+  , snippet(attribute_ifdef)
+  , snippet(attribute_ifndef)
+  , snippet(attribute_import)
+  , snippet(attribute_include)
+  , snippet(attribute_include_lib)
+  , snippet(attribute_on_load)
+  , snippet(attribute_opaque)
+  , snippet(attribute_record)
+  , snippet(attribute_type)
+  , snippet(attribute_vsn)
+  ].
+
+%%=============================================================================
+%% Include paths
+%%=============================================================================
+-spec paths_include(els_dt_document:item()) -> [binary()].
+paths_include(#{uri := Uri}) ->
```

Comment:
is this supposed to provide completion for paths in include/include_lib? That would be amazing, but I don't seem to get it to work, though.

Reply:
Yeah, that's the idea atleast :) Good catch!
It needs to be triggered right after `-include("`.
I've only tried it out on a limited number of projects on my own computer, so there could definitely be issues with the code on other platforms or projects.
When you say it doesn't work, does it hang or does it simply not find any completions?
Does it not work on the erlang_ls repo? 
May I ask to see your erlang_ls.config, so I can try to reproduce it?

Reply:
It does not find any completions for me. Tried on the `erlang_ls` code base and the Klarna one. But since I still like the idea and it does not cause any harm, I'm keen to merge this in and to eventually follow up with a fix before the next release. I will also try to clean and restart everything on my side to see if it was an artifact of my setup.

Reply:
@plux Issue confirmed. Redbugging the code, I don't seem to be able to trigger the completions.
Steps I try:
* Type `-inc`
* Select `-include_lib` from the dropdown
* Wait -> Nothing
* Start typing the name of an existing app -> Nothing
* Move the first `"` and wait -> Nothing
* Try to retype `"` -> Nothing
* Trigger a manual `completion-at-point` or `company-complete` -> Nothing

According to Redbug, the completion is not triggered at all. What should trigger it? Maybe we're missing a `"` trigger character?

Tried in both Emacs and VS Code. Which editor are you using?

Reply:
That seems odd, needs to be investigated further. :)
Currently I didn't add `"` as a trigger character, so only manually triggering the completion is intended to work.
Adding `"` as a trigger character sounds like a good idea, I will try it out.
 
I'm using Emacs with `lsp-mode` and `company`.